### PR TITLE
26 create custom json button

### PR DESCRIPTION
**Pull request by MennaHisham**

### CODE REVIEW

File: src/app/addjson/_components/alertMessage/AlertMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import Button from '@/app/_components/button/Button';
+import { useRouter } from 'next/navigation';
+
+interface AlertMessageProps {
+  onClose: () => void;
+  onConfirm: () => void;
+}
+
+const AlertMessage = ({ onClose, onConfirm }: AlertMessageProps) => {
+  //   const router = useRouter();
```
Comment (NourElturky) [inline src/app/addjson/_components/alertMessage/AlertMessage.tsx:12]: if you aren't using it please delete it

File: src/app/_components/startScreen/StartScreen.tsx [hunk 2]
```diff
@@ -1,13 +1,20 @@
-import Link from 'next/link';
```
Comment (NourElturky) [inline src/app/_components/startScreen/StartScreen.tsx:1]: "use client

File: src/app/_components/svg/NextArrow.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 export const NextArrow: React.FC<SvgProps> = ({
```
Comment (NourElturky) [inline src/app/_components/svg/NextArrow.tsx:1]: delete it

**Review by saraEid77 — APPROVED** [review]

File: src/app/_components/startScreen/StartScreen.tsx [hunk 4]
```diff
@@ -1,13 +1,22 @@
...
-      <h1 className='text-[#4CB087] font-extrabold text-5xl mt-[10%]'>Welcome to Our Questioner</h1>
-      <Link href="/questions" className='w-full flex justify-center'>
-        <button className='text-white bg-[#4CB087] rounded w-[10%] mt-[10%] text-center h-10'>Start</button>
-      </Link>
+    <div className="flex flex-col items-center space-y-10">
+      <h1 className="text-emerald-500 font-bold text-5xl mt-10">
+        Welcome to Our Questionaire
+      </h1>
+      <Button text="Start" onClick={() => handleClick('/questions')} />
+      <Button text="Custom JSON" onClick={() => handleClick('/addjson')} />
```
Comment (fadyfawzydev) [inline src/app/_components/startScreen/StartScreen.tsx:19]: use link instead of Buttons

File: src/app/success/page.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+import { useRouter } from 'next/navigation';
+
+import Button from '../_components/button/Button';
+
+const Success = () => {
+  const router = useRouter();
+
+  return (
+    <div className="flex justify-center items-center h-screen">
+      <Button text="Back to Homepage" onClick={() => router.push('/')} />
```
Comment (fadyfawzydev) [inline src/app/success/page.tsx:12]: Should be Link normally

**Review by fadyfawzydev — CHANGES_REQUESTED** [review]

**Pull request merged**